Dockerfile

```
@@ -14,3 +14,6 @@ RUN set -eux \
 COPY . .
 
 CMD ["yarn", "run", "dev"]
+
+test
```

Hadolint (2.12.0)

[error] unexpected 't'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)